### ISSUE TITLE
UITests always FAILS

### ISSUE THREAD

**Issue by DanielZanchi** [MakeAWishFoundation/SwiftyMocky#161]
I'm having a problem with the UITest target. I have the _Mock.generated.swift_ file inside the target.
my podfile is:
```
platform :ios, '12.0'

target 'FuelTracker' do
    # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
    use_frameworks!
    
    # Pods for FuelTracker
    pod 'SQLite.swift', '~> 0.11.5'
    pod 'SwiftyMocky'
    
    target 'FuelTrackerTests' do
        inherit! :search_paths
        # Pods for testing
    end
    
    target 'FuelTrackerUITests' do
        inherit! :search_paths
        # Pods for testing
        pod 'SwiftyMocky'
        pod 'Sourcery'
    end
end
```

When I try to run the tests inside the _FuelTrackerUITests_ target I get this error only if the _Mock.generated.swift_ file is there:

> FuelTrackerUITests-Runner.app (32924) encountered an error (Early unexpected exit, operation never finished bootstrapping - no restart will be attempted. (Underlying error: The test runner failed to load the test bundle. Executable fails due to linking.))

The other target _FuelTrackerTests_ works fine.
Anyone can help?

**Comment by amichnia** [comment]
@Danny182 Hi, I will look into it on weekend. We are just before release, so there is high chance it will be shipped next week with 3.1

Stay tuned :)

**Comment by amichnia** [comment]
@DanielZanchi 

1. In the main App target use 'SwiftyMocky/Prototyping' subspec. The case is that SwiftyMocky uses XCTest by default, which is not accessible from the main app. That's why we have prototyping subspecs.

2. It seems this linker error is not SwiftyMocky issue. In general, you cannot use/subclass/adopt any code from the main app from the UI tests target. I tried to create an instance of some class from app target in UITests (without any dependencies whatsoever) - behavior is exactly the same.

[
https://stackoverflow.com/questions/33755019/linker-error-when-accessing-application-module-in-ui-tests-in-xcode-7-1](https://stackoverflow.com/questions/33755019/linker-error-when-accessing-application-module-in-ui-tests-in-xcode-7-1)

So far, the only workaround for that was having files added to both App and UITests targets, but I don't know if that will work in your case.
